user-docs/latest/getting-started/introduction.md

```diff
@@ -1,3 +1,65 @@
+
+Before going any further into this user guide, we recommend you to acknowledge the main concepts below. They will help you to understand how Strapi works, and ensure a smooth Strapi experience.
+
+- **Development, Staging or Production Environment** <br> When starting working on your application, it is in a development environment, which is the status for the content structuration and application configuration phase. After deploying your application, it is in production or staging. This status change impacts how you can use your Strapi application, as some features are only available in development environment, such as the Content-Types Builder. In the user guide, the availability or not of a feature, depending on the application status, is always mentioned in the feature's introduction.
+
+- **Versions** <br> Strapi is constantly evolving and growing. This implies that new releases are quite frequent, to improve what is already available but also to add new features to Strapi. For every new Strapi version, we communicate through our main channels and by sending notifications both on your terminal (when launching your Strapi application), and on your application's admin panel. We always recommend to use the latest version, especially since our documentation, including this user guide, are only aligned with the latest<!-- (see Update Strapi version or refer to our migration guides to update your Strapi application) -->. If you are on an older version of Strapi, some information in this documentation may not apply to your application. 
+
+- **Roles and Permissions** <br> Some features of the admin panel, as well as the content managed with Strapi itself, are ruled by a system of permissions. From your Strapi admin panel, you have the possibility to define, at a detailed level, the roles and permissions of all administrators and end-users. In this user guide, all features and possible options are documented. It is however possible, depending on your role and permissions, that you may not be able to access all these features and options. In that case, please refer to the main Super Admin of your Strapi application.
+
+With all this in mind, you should be ready to start your Strapi experience!
+
+
+## Accessing the admin panel
+
+The admin panel is the back office of your Strapi application. From the admin panel, you will be able to manage content types, and write their actual content. It is also from the admin panel that you will manage users, both administrators and end-users of your Strapi application.
+
+::: tip IMPORTANT
+In order to access the admin panel, your Strapi application must be launched<!-- (see Installing Strapi) -->, and you must be aware of its URL.
+:::
+
+![User profile](../assets/getting-started/login-page.png)
+
+To access the admin panel:
+
+1. Go to the URL of your Strapi application.
+2. (optional) Click on the **Open the administration** button.
```

Comment:
Only applies to development, not sure if you have some method to point this out quickly